pkg/debug/scripts/collect-info.sh

```diff
@@ -279,6 +279,7 @@ dmidecode        > "$DIR/dmidecode"
 ls -lRa /dev     > "$DIR/ls-lRa-dev"
 ls -lRa /persist > "$DIR/ls-lRa-persist"
 free             > "$DIR/free"
+du -h            > "$DIR/du-h"
```

Comment:
Did you really want `du`? `du` will calculate the disk usage of the current path (or the path provided as argument)... I guess you were looking for df (disk free command), right?

Reply:
Yeah, you are right. Fixed. Although I think du would be also useful )